src/domain/common/utils/utils.ts

```diff
@@ -1,6 +1,7 @@
 export function getNumberString(value: number): string {
   // Prevent scientific notation
-  return value.toLocaleString('fullwide', {
```

Comment:
`'fullwide'` is not a standard locale identifier. This seems to be a bug in the original implementation, which JS is probably ignoring and therefore defaulting to the host locale.